src/main/java/uk/gov/companieshouse/authcodenotification/service/PrivateDataRetrievalService.java

```diff
@@ -16,26 +16,26 @@
 @Service
 public class PrivateDataRetrievalService {
 
-    private static final String OVERSEAS_ENTITY_URI_SECTION = "/overseas-entity/%s/entity-data";
+    private static final String OVERSEAS_ENTITY_URI_SECTION = "/company/%s/registered-email-address";
 
     @Autowired
     private ApiClientService apiClientService;
 
-    public OverseasEntityDataApi getOverseasEntityData(String requestId, String companyNumber)
+    public RegisteredEmailAddressJson getOverseasEntityData(String requestId, String companyNumber)
             throws ServiceException {
         var logDataMap = new DataMap.Builder().companyNumber(companyNumber).build();
         try {
             ApiLogger.infoContext(requestId, "Retrieving overseas entity data from database", logDataMap.getLogMap());
 
-            var overseasEntityDataApi = apiClientService
+            var registeredEmailAddress = apiClientService
```

Comment:
Is this use of var here and on line 26 to satisfy sonarqube? I really think we should not be losing type information - this is not a new change, but just clarifying. I will try to get the sonarqube rules updated if so.

Reply:
Ok thanks - I will chase the rule update as I feel this is just making code hard to follow in terms of typing information.